### PR TITLE
Ghelman87 patch 1

### PR DESCRIPTION
**Pull request by ghelman87**
This PR

* [x] 
* [ ] 
* [ ] 

Follows #.
Related to #.
Fixes #.

### CODE REVIEW

File: src/Security/Core/User/EuloginUserProvider.php [hunk 1]
```diff
@@ -7,16 +7,27 @@
...
 use Symfony\Component\Security\Core\User\UserInterface;
 
 use function get_class;
+use Symfony\Component\Security\Http\EntryPoint\AuthenticationEntryPointInterface;
 
 /**
  * Class EuloginUserProvider.
  */
-class EuloginUserProvider implements EuloginUserProviderInterface
+class EuloginUserProvider implements EuloginUserProviderInterface, AuthenticationEntryPointInterface
```
Comment (drupol) [inline src/Security/Core/User/EuloginUserProvider.php]: This is a no-go. The AuthenticationEntryPointInterface should be set on a Guard, not on a UserProvider.

Comment (ghelman87) [reply src/Security/Core/User/EuloginUserProvider.php]: You can live it. only commit  bf0b5c8219d9defbd4eebfc6a823ff2905a6b6e0 is interesting at this moment
If I would have access to console ...

File: Resources/config/packages/dev/eulogin_cas_security.yaml [hunk 2]
```diff
@@ -5,4 +5,6 @@ security:
     firewalls:
         default:
             guard:
-                provider: eulogin
\ No newline at end of file
+                authenticators:
```
Comment (drupol) [inline Resources/config/packages/dev/eulogin_cas_security.yaml]: Those 2 first lines needs to be added in cas-bundle, and I just did it.

File: src/Security/Core/User/EuloginUserProvider.php [hunk 3]
```diff
@@ -31,7 +31,7 @@ public function loadUserByResponse(ResponseInterface $response): CasUserInterfac
     /**
      * {@inheritdoc}
      */
-    public function loadUserByUsername(string $username)
+    public function loadUserByUsername($username)
```
Comment (drupol) [inline src/Security/Core/User/EuloginUserProvider.php]: This is already done in eulogin-bundle branch 4.4.

File: src/Security/Core/User/EuloginUserProvider.php [hunk 4]
```diff
@@ -51,7 +51,7 @@ public function refreshUser(UserInterface $user)
     /**
      * {@inheritdoc}
      */
-    public function supportsClass(string $class)
+    public function supportsClass($class)
```
Comment (drupol) [inline src/Security/Core/User/EuloginUserProvider.php]: This is already done in eulogin-bundle branch 4.4.

**Comment by drupol** [comment]
Dear @ghelman87,

I updated drupol/cas-bundle to fix the behavior, let me know how it goes, after your holidays it's fine as well :-)

Enjoy your holidays!

**Comment by ghelman87** [comment]
@drupol You too
You will have one less person to disturb You :)

**Comment by drupol** [comment]
It's fine no worry :-)

**Pull request closed without merging**